pkg/apis/kops/componentconfig.go

```diff
@@ -142,6 +142,10 @@ type KubeletConfigSpec struct {
 	SystemReservedCgroup string `json:"systemReservedCgroup,omitempty" flag:"system-reserved-cgroup"`
 	// Enforce Allocatable across pods whenever the overall usage across all pods exceeds Allocatable.
 	EnforceNodeAllocatable string `json:"enforceNodeAllocatable,omitempty" flag:"enforce-node-allocatable"`
+	// RuntimeRequesTimeout is timeout for runtime requests on - pull, logs, exec and attach
+	RuntimeRequesTimeout *metav1.Duration `json:"RuntimeRequestTimeout,omitempty" flag:"runtime-request-timeout"`
```

Comment:
Lower case on Runtime JASON value please

Reply:
JSON value ..

Reply:
good spot! ... a copy and paste error .. i'll fix it now :-)